docs/pages/includes/config-reference/database-config.yaml

```diff
@@ -191,6 +191,12 @@ db_service:
       spn: MSSQLSvc/ec2amaz-4kn05du.dbadir.teleportdemo.net:1433
       # Optional path to Kerberos configuration file. Defaults to /etc/krb5.conf.
       krb5_file: /etc/krb5.conf
+      # Name of the service account Teleport uses to perform LDAP queries for retrieving user SIDs.
+      # Required for PKINIT if Active Directory expects the user SID to be embedded in the certificate.
```

Comment:
I would drop the `if Active ...` part to make it more serious. We suppose to have these years ago =D

Reply:
Dropped, as you say this is more of a default right now so no point in confusing people. I can imagine answering the question "How do I check if this applies to my env?" would be somewhat involved.